package.json

```diff
@@ -46,10 +39,22 @@
   },
   "devDependencies": {
```

Comment:
It happens automatically on npm install. They are only not installed on npm install --production apparently.